docs/src/components/document-renderer.tsx

```diff
@@ -14,7 +14,7 @@ export default async function DocumentRenderer({
   document,
 }: DocumentRendererProps & { slug: string }) {
   const highlighter = await shiki.getHighlighter({
-    theme: 'min-light',
+    theme: 'github-light',
```

Comment:
This theme improves the comment contrast but imo is a bit dull.

Not necessarily for this PR, but what do you think about making our own theme? 

You can find an existing theme as a base in `docs/node_modules/shiki/themes` and copy it into the repo then start tweaking values. 

You load it like this:
```
const highlighter = await shiki.getHighlighter({
    theme: JSON.parse(
      fs.readFileSync('./src/styles/keystatic-theme.json', 'utf-8')
    ),
  });
```

And get beautiful colours like this: 😂
<img width="720" alt="CleanShot 2023-06-08 at 09 35 42@2x" src="https://github.com/Thinkmill/keystatic/assets/5933897/5fbfba06-1fd2-493e-9dd0-3ad1a25fb7ff">

Reply:
Niice yeah I couldn't figure out how to do that yesterday arvo and didn't notice much difference between `min-light` and `github-light` except for `// comment` color, and also thought that the github highlighting is a solid universal/classic theme, but it would make sense to customise it a bit to make it our own if it seems too dull 🤔

Reply:
I could be judging too harshly, possibly cause the field APIs are a snippet of an object which doesn't get much highlighting. 

Happy to leave this for now.